modules/core/shared/src/main/scala/lucuma/core/geom/gmos/GmosCandidatesArea.scala

```diff
@@ -39,5 +40,36 @@ trait GmosCandidatesArea {
       a <- posAngles
       o <- offsetPositions
     } yield candidatesAreaAt(a, o)).fold(ShapeExpression.Empty)(_ ∪ _)
+
+  /**
+    * GMOS inner radius
+    */
+  def innerRadius(probe: GuideProbe): ShapeExpression =
```

Comment:
This is about avoiding selection of the science target itself or avoiding vignetting?  It seems to be the former in the case of GMOS OI and the later for PWFS?

Reply:
One reason I ask is because I'm hoping to not have the vignetting avoidance inner radius at all for PWFS and instead define the geometries and vignetting as we do for GMOS OI.

Reply:
If it is just about avoiding the selection of the science target, why would it differ based on guide probe?

Reply:
I suppose not only the science target but those very near. This was specified as 20 arcsecs for GMOS OIWFS